pages/bitmask.rst

```diff
@@ -11,7 +11,7 @@
 .. _`MzLS`: ../mzls
 .. _`DESI`: http://desi.lbl.gov
 .. _`DR3`: ../dr3
-.. _`files page`: ../dr4/files
+.. _`files page`: ../dr5/files
 .. _`legacypipe`: https://github.com/legacysurvey/legacypipe
 
 The ``ccds-`` and ``survey-`` files described on the `files page`_ contain a column ``bitmask``, which is defined as:
```

Comment:
I'd like to check the bitmask page, but I can't find which page links to bitmask.rst.

Reply:
There was a link to bitmask.rst in dr4/files.rst, but that seems to be missing from dr5/files.rst.

Reply:
Good catch. I've specifically linked this back to the DR4 files page, as you noted, `bitmask` does not appear in the DR5 data model.